examples/src/bin/server_acceptor.rs

```diff
@@ -117,8 +117,14 @@ fn main() {
         // connection.
         let accepted = loop {
             acceptor.read_tls(&mut stream).unwrap();
-            if let Some(accepted) = acceptor.accept().unwrap() {
-                break accepted;
+
+            match acceptor.accept() {
+                Ok(Some(accepted)) => break accepted,
+                Ok(None) => continue,
+                Err((e, mut alert)) => {
+                    alert.write(&mut stream).unwrap();
+                    panic!("error accepting connection: {}", e);
+                }
```

Comment:
I think we need the same again on line ~136

Reply:
Good catch, thanks. Fixed